Governance/publish/publish-intro.md

```diff
@@ -1,14 +1,15 @@
 # Guidelines to publish climate data
 
-## Scope of the guidelines
-This guidelines are covering the main steps involved in publishing a climate dataset.
-They illustrate how to prepare a dataset for publication, what you want to include both in the dataset itself and in the metadata that will be created for the publication. What are the common requirement for a dataset to publication ready and what are th specific requirements based on the publisher. As this is looking at the Australian context we focused on the institutions we know of that publish climate data. If you are aware of others we didn't covert please contact us by creating an issue.
+These guidelines are covering the main steps involved in publishing a climate dataset.
+We tried to list the publishing options available to climate researchers in Australia, based on their organisation and the type of data to publish. 
+We then covered some of these options in detail, highlighting how to prepare a dataset for publication, what to include both in the dataset itself and in the metadata record.
+We focused on institutions and projects we know of that publish climate data. If you are aware of others that we didn't cover please contact us by creating an issue.

```

Comment:
link [creating an issue](https://github.com/ACDguide/Governance/issues/new/choose) in this sentence.

Reply:
about the journal/dataset examples I agreed they should either both in the appendix or at the end of this chapter, which might make more sense, but they're basically equally relevant. And I'll add the link, thanks for spotting that

Reply:
Yep I'd be happy for it to be moved in here (ie update toc), feel free to add it to this PR if possible.